### ISSUE TITLE
注册时bcrypt加密

### ISSUE THREAD

**Issue by jj-apm** [qwe6802557/fur-admin#3]
请问你的注册时是使用bcrypt进行加密的吗

**Comment by qwe6802557** [comment]
是的 由于安装bcrypt插件上传github会出问题 改用了bcrypt.js插件 可以自行在dependence依赖中删除bcrypt 这样npm install 就不会再安装了

**Comment by jj-apm** [comment]
但是你用的是this.app.encryptPw(FACTORY_STRENTTH,password)进行加密的啊

**Comment by qwe6802557** [comment]
这是egg的extend拓展方法 在extend下的app 和 helper文件里能找到封装的方法

**Comment by jj-apm** [comment]
好的，谢谢

**Comment by qwe6802557** [comment]
项目已全部完成 不过数据库需要自行创建

**Comment by jj-apm** [comment]
请问这个系统是有做SSR服务端渲染的吗？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"qwe6802557"<notifications@github.com&gt;;
发送时间:&nbsp;2020年4月15日(星期三) 凌晨0:56
收件人:&nbsp;"qwe6802557/fur-admin"<fur-admin@noreply.github.com&gt;;
抄送:&nbsp;"16级网络设计与管理江俊杰"<1431105276@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [qwe6802557/fur-admin] 注册时bcrypt加密 (#3)





 
项目已全部完成 不过数据库需要自行创建

**Comment by qwe6802557** [comment]
前后端分离模式哦 非ssr
